ts/features/zendesk/screens/ZendeskChooseCategory.tsx

```diff
@@ -39,24 +37,17 @@ export type ZendeskChooseCategoryNavigationParams = {
   assistanceForPayment: boolean;
 };
 
-type Props = {
-  navigation: CompatNavigationProp<
-    IOStackNavigationProp<ZendeskParamsList, "ZENDESK_CHOOSE_CATEGORY">
-  >;
-};
+type Props = IOStackNavigationRouteProps<
+  ZendeskParamsList,
+  "ZENDESK_CHOOSE_CATEGORY"
+>;
 
 /**
  * this screen shows the categories for which the user can ask support with the assistance
  */
 const ZendeskChooseCategory = (props: Props) => {
   const dispatch = useDispatch();
-  const navigation =
-    useNavigation<
-      IOStackNavigationProp<ZendeskParamsList, "ZENDESK_CHOOSE_CATEGORY">
-    >();
-  const assistanceForPayment = props.navigation.getParam(
-    "assistanceForPayment"
-  );
+  const assistanceForPayment = props.route.params.assistanceForPayment;
```

Comment:
```suggestion
  const { assistanceForPayment } = props.route.params;
```
Maybe more clear, what do you think?